### PR TITLE
[ONNX] Provide support for function in ONNX models. 

### PR DESCRIPTION
**Pull request by Ewa21**

### CODE REVIEW

File: src/ngraph/frontend/onnx_import/core/graph.cpp [hunk 1]
```diff
@@ -93,11 +98,39 @@ namespace ngraph
...
+            // DEBUG
+            for (const auto& node : m_graph_proto.node())
+            {
+                std::cout << node.op_type() << std::endl;
+            }
+            std::string data;
+            m_graph_proto.SerializeToString(&data);
+            std::cout << data;
+
+            // END DEBUG
```
Comment (tsocha) [inline src/ngraph/frontend/onnx_import/core/graph.cpp]: This code should be removed before making a final PR.

File: cmake/external_onnx.cmake [hunk 2]
```diff
@@ -32,6 +32,7 @@ set(ONNX_GIT_BRANCH rel-${ONNX_VERSION})
 
 add_definitions(-DONNX_BUILD_SHARED_LIBS=ON)
 add_definitions(-DONNX_ML=ON)
+add_definitions(-DONNX_NAMESPACE=onnx)
```
Comment (postrational) [inline cmake/external_onnx.cmake]: We should customize this namespace, not use the default `onnx`. This should be done in a separate PR.

File: src/ngraph/frontend/onnx_import/op/round.cpp [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+namespace ngraph
+{
+    namespace onnx_import
+    {
+        namespace op
+        {
+            namespace set_1
+            {
+                NodeVector round(const Node& node)
```
[the remark this answers is not in the document]
Comment (Ewa21) [reply src/ngraph/frontend/onnx_import/op/round.cpp]: DynamicQuantizeLinear op uses Round op.

File: src/ngraph/frontend/onnx_import/core/graph.cpp (hunk 1 above)
Comment (Ewa21) [reply src/ngraph/frontend/onnx_import/core/graph.cpp]: Done

File: test/onnx/onnx_import.in.cpp [hunk 4]
```diff
@@ -1904,3 +1904,67 @@ NGRAPH_TEST(onnx_${BACKEND_NAME}, model_reciprocal)
 
     test_case.run();
 }
+
+NGRAPH_TEST(onnx_${BACKEND_NAME}, model_dynamic_quantize_linear)
+{
+    const auto dql_fn = onnx_import::import_onnx_model(
+        file_util::path_join(SERIALIZED_ZOO, "onnx/dynamicquantizelinea.onnx"));
```
[the remark this answers is not in the document]
Comment (Ewa21) [reply test/onnx/onnx_import.in.cpp]: Done

**Comment by postrational** [comment]
Please Update branch with current master.

**Comment by postrational** [comment]
Please convert test models to prototxt format.

File: src/ngraph/frontend/onnx_import/core/model.hpp [hunk 5]
```diff
@@ -47,6 +47,20 @@ namespace ngraph
                 return m_model_proto->producer_version();
             }
 
+            std::int64_t get_operator_set_id(const std::string& domain) const
```
Comment (postrational) [inline src/ngraph/frontend/onnx_import/core/model.hpp]:
```suggestion
            std::int64_t get_operator_set_version(const std::string& domain) const
```

**Comment by postrational** [comment]
You should move`Round` to a separate PR and write some tests for it.
At least test the documented cases:
```
round([0.9]) = [1.0]
round([2.5]) = [2.0]
round([2.3]) = [2.0]
round([1.5]) = [2.0]
round([-4.5]) = [-4.0]
```

File: src/ngraph/frontend/onnx_import/core/graph.hpp [hunk 6]
```diff
@@ -56,14 +56,17 @@ namespace ngraph
...
-            const onnx::GraphProto* m_graph_proto;
+            onnx::GraphProto m_graph_proto;
             std::vector<Node> m_nodes;
             std::vector<ValueInfo> m_inputs;
             std::vector<ValueInfo> m_outputs;
             ParameterVector m_parameters;
             std::map<std::string, std::shared_ptr<ngraph::Node>> m_ng_node_cache;
             std::map<std::string, Tensor> m_initializers;
             Model* m_model;
+            const std::set<std::string> function_to_expand_whitelist = {"DynamicQuantizeLinear"};
```
Comment (tsocha) [inline src/ngraph/frontend/onnx_import/core/graph.hpp]: NON-BLOCK
```suggestion
            const std::set<std::string> expand_function_whitelist = {"DynamicQuantizeLinear"};
```

**Comment by Ewa21** [comment]
Move to the  OpenVino repository.
